src/implementations/twig/components/gallery/gallery-item.html.twig

```diff
@@ -119,15 +127,19 @@
         </div>
       {% endif %}
     {% endif %}
-    {% if _item.video is defined and _item.video is not empty %}
+
+    {% if _item.embedded_video is not empty %}
+      {# Embedded video #}
+      <img src="{{ _thumbnail }}" alt="{{ _picture.img.alt|default('') }}" class="ecl-picture ecl-gallery__thumbnail ecl-gallery__image" />
```

Comment:
mmh...i'm thinking that this might break something on the application side, it is not possible to pass custom attributes to this img tag and i think it's required on ewpp side, probably for analytics, so we could go back to the object for the thumbnail, or we could add another param where to pass those attrs and then process them in the template, annoying but i think it's needed